llvm/utils/lit/tests/shtest-umask.py

```diff
@@ -0,0 +1,18 @@
+# Check the umask command
+
+# RUN: not %{lit} -a -v %{inputs}/shtest-umask | FileCheck -match-full-lines %s
+
+# CHECK: -- Testing: 3 tests{{.*}}
+
+# CHECK-LABEL: FAIL: shtest-umask :: umask-bad-arg.txt ({{[^)]*}})
+# CHECK: umask bad
+# CHECK: # | Error: 'umask': invalid literal {{.*}}
+
+# CHECK-LABEL: FAIL: shtest-umask :: umask-too-many-args.txt ({{[^)]*}})
+# CHECK: umask 0 0
+# CHECK: # | 'umask' supports only one argument
+
+# CHECK: Total Discovered Tests: 3
+# CHECK: {{Passed|Unresolved}}: 1 {{\([0-9]*\.[0-9]*%\)}}
```

Comment:
```suggestion
# CHECK: {{Passed|Unsupported}}: 1 {{\([0-9]*\.[0-9]*%\)}}
```
"Unresolved" means something else.

Reply:
Yes, I think you're right, but then I don't understand why CI testing for this PR passed on Windows.

Reply:
Does CI run the lit tests? I downloaded the log and I don't see "shtest-umask" in it.

Reply:
I see, thanks, it runs `ninja check-clang check-clang-tools check-lld check-llvm check-mlir check-polly` but that does not include `check-lit` :-(

Reply:
Working on getting the test to work in my environment. Taking longer than I expected.